mockup-contributions_list.php

```diff
@@ -0,0 +1,922 @@
+<?php
```

Comment:
I was thinking everyones designs would eventually be merged into master, so we can show them all off via the index page. And so I can fetch from master and put onto Heroku easily. But I really wanted to do folders and/or separate mockups from templates later down the line. Let me know if you agree.